lib/notesclub_web/live/notebook_live/show.html.heex

```diff
@@ -28,6 +28,9 @@
             <%= img_tag static_path(@socket, "/images/github-logo.svg"), class: "h-8 rounded-full", alt: "avatar" %>
           </.link>
         </div>
+        <div class="pt-3 pl-4">
```

Comment:
If we increase the size of the icon, we need to change the padding so it is properly aligned vertically:
```suggestion
        <div class="pt-2 pl-4">
```